### ISSUE TITLE
Ionic Build and Run Android resulting in EACCES error since npm install -g ionic@2.0.0-beta.32

### ISSUE THREAD

**Issue by sarahhostyk** [driftyco/ionic-cli#1211]
I tried running sudo npm install -g ionic@beta which upgraded me to beta 33 and caused a bunch of problems so I did npm install -g ionic@2.0.0-beta.32.

Now I can't build or run android. I get an EACCES error. I've tried sudo and also running `sudo chown -R  me /Users/me/Documents/app folder `but still doesn't work.

Please see error below.

![screen shot 2016-07-26 at 11 46 02 pm](https://cloud.githubusercontent.com/assets/19256663/17163655/84258df8-538f-11e6-91c3-62b9af8cf1ee.png)

**Comment by jthoms1** [comment]
Found the issue. There is a problem with npm where you need to always publish packages with bundledDependencies from npm2 with the node_modules structure of npm2 (nested). When I released ionic@2.0.0-beta.33 I was using npm3 with the flat structure. So release beta.33 would only work with npm3 installs, which is what I used for testing.

I am sorry for the pain this might have caused. It is resolved in beta.35. I have tested and we should be ok now. Please let me know if you are still having issues. Thank you!

Fix: npm install -g ionic@2.0.0-beta.35

**Comment by sarahhostyk** [comment]
@jthoms1 Thanks! I upgraded and it got rid of those errors. But since upgrading to 33 and then 35 I am getting really stuck on building both android and ios, especially with many EACCES errors (I am the owner).

I ran `sudo chown -R $(whoami) ~/desktop/pt/platforms/android/gradle/wrapper` which got rid of them for android but resulted in this build failure?

![screen shot 2016-07-28 at 11 24 17 am](https://cloud.githubusercontent.com/assets/19256663/17218818/314ce6e8-54b7-11e6-9375-662e1d8e0492.png)

However I can't get rid of them for iOS. I tried `sudo chown -R $(whoami) ~/desktop/pt/platforms/ios/www/build` but still get 100s of these EACCES errors when trying to build. 

![screen shot 2016-07-28 at 11 10 20 am](https://cloud.githubusercontent.com/assets/19256663/17218910/636801ee-54b7-11e6-91e7-1ff003c4aa7b.png)

I didn't have a problem building or running Android or iOS before upgrading. Thank you for your help!
